src/main/java/org/jenkinsci/plugins/EnvironmentHelper.java

```diff
@@ -0,0 +1,89 @@
+package org.jenkinsci.plugins;
+
+import hudson.model.Hudson;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import hudson.slaves.NodeProperty;
+import hudson.slaves.NodePropertyDescriptor;
+import hudson.util.DescribableList;
+import org.apache.commons.lang.StringUtils;
+
+import javax.annotation.CheckForNull;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * Created by Leo on 4/20/2016.
+ * Helper to work with environment variables.
+ */
+public class EnvironmentHelper {
+    private static final Logger LOGGER = Logger.getLogger(EnvironmentHelper.class.getName());
+
+	public static final String BuildDisplayNameVar = "NEW_BUILD_NAME";
+	
+    public static void SetEnvironmentVariable(@CheckForNull String key, @CheckForNull String value, PrintStream logger) {

```

Comment:
in Java, methods names are in camelCase. Other methods in build-name-setter-plugin are also in camelCase